### DISCUSSION TITLE
사전 메일 관련 논의

### DISCUSSION THREAD

**Discussion by dongho108** [woowacourse-teams/2022-ternoko#105] [Show and tell]
# 사전 메일 발송 시간

- 5시 + 11시 두번에 나누어 보낸다.
    - 장점 : 코치의 워크타임에 보낸다는 점
    - 단점 :
        - 코치가 두번 확인해야함(5시,11시 59분)
        - 만약 아래에 3번정책과 같이 갈 경우 메일 전송에 관련된 상태 추가 필요
            
            → 5시에 보낸게 11시 59분에 한번 더 안가게 하기 위해서 
            
- 11시 59분에 일괄로 보낸다.
    - 장점 : 코치가 한번만 확인해도 됨.
    - 단점 : 코치가 당일 되어서야 사전 메일을 확인할 수 있음 (전날 11:59분에 보내니까)

# 코치가 사전질문내용을 볼 수 있는지?

- 메일 보내기 전까지 편집 + 코치는 상시 확인
- 메인 보내기 전까지 편집 + 코치는 하루 전부터 확인
- 메일 보내기 전까지 편집 + 확정버튼 추가해서 확정버튼 누르면 편집 불가 및 코치 확인 가능
    - 확정하면 인터뷰상태 변환(editable → fix)하는 로직 필요.
- 메인 보내기 전까지 편집 + 코치는 당일 아침에 확인
    

    
- 여기서 사전메일 내용 빠지고 시간하고 크루이름만 표시
<img width="474" alt="스크린샷 2022-07-27 21 02 52" src="https://user-images.githubusercontent.com/54317630/181242199-7af9c879-3508-4983-b14c-2c9f61d5b7d3.png">

# 수정 논의

- 수정 가능 + 상시 확인 가능
    - 미정된 내용을 계속 보게됨
- 수정 가능 + 확인 불가
    - 어?!
    - 고민점 :
    - 코치들이 면담전날 사전질문 내용을 확인하고 있었을까?
    - 코치들은 과연.. 우리의 사전질문을.. 면담날짜 훨씬 전에 읽어볼까..?
    - 오히려 면담직전에 한번 읽어보고 오지않을까 싶네여
    - + 브라운 가라사대 코치들은 사실 일정을 보고싶음 사전메일도 메일의 내용이 아닌 시간적 확정을 하기 위해서임.
- 수정 불가능
    - 크루입장에서 불편

**Comment by dongho108** [comment]
제이슨의 의견은 이렇습니다..!
<img width="690" alt="스크린샷 2022-07-27 20 53 37" src="https://user-images.githubusercontent.com/54317630/181240447-a9bac749-ff2d-4ac9-b06c-5131827a28f5.png">

**Comment by her0807** [comment]
애쉬 ~ 메일 관련해서 많은 고민을 하셨네요 👍🏻✨

이런 디테일이 서비스의 품질을 좌우 하는 것 같아요. 정말 감사합니다 😇

## 1) 메일 발송은 11시 59분
메일 발송은 현재 정책상 진행하는 것이기 때문에 11시 59분에 보내는게 맞다고 생각합니다. 

## 2) 면담 사전 질문 확인은 상시로 !
코치님들이 메일로 확인을 못하는 부분을 터놓고 사이트에 들어와서 상시로 가능하기 때문에 리텐션 측면에서도 좋다고 봅니다!

![스크린샷 2022-07-27 오후 8 54 37](https://user-images.githubusercontent.com/26570275/181240632-6fdda168-0184-4b8d-bf5b-d5455f7f22a9.png)

**Comment by her0807** [comment]
추가 디스커션 https://www.notion.so/4b8875d06e104da48efc25b25dc5d26c

**Comment by Juhyung990122** [comment]
임시저장 - 코치에게 안보임, 수정 가능(내용적인 수정 )
제출 - 코치에게 보임, 수정 가능(오타, 가벼운 수정)
메일은 11시 59분에 일괄로 전송

**Comment by dongho108** [comment]
### 종합의견

### 사전질문내용 작성 및 코치분들의 권한
크루가 예약할 때 사전질문내용을 작성하는 것이 맞다.
이유 : 사실 면담을 위해 코치들의 시간을 예약한다는 행위 자체가 무거운 행위인데, 크루의 편의를 위해 시간예약을 먼저 할 수 있게 하는 것은 원래 방식이 잘못된 행위였다. 코치분들의 편의를 위해 질문내용들은 예약할때 작성하도록 하는것이 맞다.

### 메일전송
메일전송은 전날 23:59에 일괄전송한다.
추후에 코치분들로부터 불편하다는 의견이 나온다면 수정해야할 수 있다.
